fastdtw.pyx

```diff
@@ -0,0 +1,135 @@
+#!python
```

Comment:
It's customary for cython modules to start with an underscore. I.e. Python implementation in fastdtw.py, Cython implementation in _fastdtw.pyx (and _fastdtw.so). You can then make the fast version preferred by adding at the bottom of fastdtw.py:
```py
def fastdtw(...):
    ... # python implementation

try:
    from ._fastdtw import fastdtw  # overrides above fastdtw
except ImportError:
    pass  # Compiled library not available. Perhaps issue a user warning.
```